runtime/server_http_request_test.go

```diff
@@ -136,7 +136,7 @@ func TestDoubleParseQueryValues(t *testing.T) {
 		"GET", "/foo", http.HandlerFunc(zanzibar.NewRouterEndpoint(
 			bgateway.ActualGateway.ContextExtractor,
 			deps,
-			"foo", "foo",
+			"foo", "foo", "/foo?%gh&%ij",
```

Comment:
I understand what this test for the specific url, but it is wired to say "this endpoint handles requests with the url that has the specific query values". Usually we only specify the url path, i.e., without the ? part.

Reply:
This was a mistake on my end. In its place is supposed to be just `"/foo"`